src/ain.ts

```diff
@@ -43,7 +47,7 @@ export default class Ain {
    * @param {string} providerUrl
    */
   setProvider(providerUrl: string, chainId?: number) {
```

Comment:
how about getting new AxiosConfig when setting a new provider in optional?

Reply:
Fixed!